.travis.yml

```diff
@@ -3,7 +3,9 @@ dist: xenial
 language: node_js
 node_js: '12'
 
-script: npm run build
+script: 
+  - scripts/test.sh
```

Comment:
why not have it inside here?

Reply:
Can these run in parallel? Imho it will save some time. Both can fail the build at any time.
Though usually the static checks should run first, but often single space fails to show you a real problem. So IMHO i would suggest parallel

https://docs.travis-ci.com/user/build-stages/#how-to-define-build-stages

Reply:
I think it doesn't matter for our use-case